CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 project(uranium NONE)
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.2.0)
```

Comment:
Our CentOS build VM is using CMake 2.8.12. This change would break that VM. As far as I can tell, there is no need to bump the version requirement.